config-provisioning/src/main/java/com/yahoo/config/provision/ContainerImage.java

```diff
@@ -11,14 +11,14 @@
  *
  * @author mpolden
  */
-public class DockerImage {
```

Comment:
This renaming breaks the API. Not sure you can do it, maybe by duplicating and subtyping?